doc/man1/s_client.pod

```diff
@@ -209,6 +220,34 @@ be used.
 
 The private format to use: DER or PEM. PEM is the default.
 
+=item B<-cert_chain>
+
+A file containing trusted certificates to use when attempting to build the
+client certificate chain when certificate specified via B<-cert> option is in
+use.
+
+=item B<-build_chain>
+
+Specify whether the application should build the certificate chain to be
+provided to server.
+
+=item B<-xkey infile>, B<-xcert infile>, B<-xchain>
+
+Specify an extra certificate, private key and certificate chain, these behave
+in the same manner as the B<-cert>, B<-key> and B<-cert_chain> options.  When
+specified, the callback returning the first valid chain will be in use by
+client.
+
+=item B<-xchain_build>
+
+Specify whether the application should build the certificate chain to be
+provided to client for the extra certificates provided via B<-xkey infile>,
+B<-xcert infile>, B<-xchain> options.
+
+=item B<-xcertform PEM|DER>, B<-xkeyform PEM|DER>
+
+Extra certificate and private key format and passphrase respectively.
```

Comment:
passphrase? Did you miss an option from the list?

Reply:
No. Cut'n'paste'error, though it may have sense to add -xpass option similar to the -pass option.